specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2020-05-01-preview/examples/Operations_List.json

```diff
@@ -0,0 +1,154 @@
+{
```

Comment:
The example format is wrong. Pls fix this example format.

Reply:
fixed it